src/structuralFunctionality/language-switcher/LanguageSwitcher.tsx

```diff
@@ -234,9 +234,11 @@ export default class LanguageSwitcher extends Widget {
 
     const [fieldName] = IDs;
     const currentValue = withinConfigurationExperience
-      ? config?.draft?.[fieldName]
+      ? config?.draft?.[fieldName] ?? config?.[fieldName]
       : config?.[fieldName];
 
+    if (currentValue === undefined || currentValue === null) return;
```

Comment:
this could be shortened to `if(currentValue == null) return` and it would be true for when currentValue is undefined as well

Reply:
done